setup.py

```diff
@@ -21,9 +21,11 @@
     'isort>=4.3.3',
     'pydocstyle>=2.0.0',
     'pytest-cov>=2.5.1',
+    'pytest-mock>=1.6.0',
     'pytest-pep8>=1.0.6',
     'pytest-invenio>=1.0.5',
-    'invenio-accounts>=1.1.1,<1.2.0'
+    'invenio-accounts>=1.1.1,<1.2.0',
+    'Sphinx>=1.5.1',
```

Comment:
I understand sphinx gets called in the tests. But is required in `docs`. This is a common issue accross all Invenio. We should start fixing it in all modules. However, Im not in favor of setting a second dependency here. Maybe we can extract it to a variable or append the `tests` array with `docs` (to avoid inconsistencies).

Reply:
Agreed